context/app/static/js/components/entity-search/SearchWrapper/numericFacetsProps.json

```diff
@@ -0,0 +1,303 @@
+            "max": 4680,
+            "min": 0,
+            "interval": 234
+        },
+        "metadata.metadata.resolution_y_value": {
+            "max": 4659,
+            "min": 0,
+            "interval": 232
+        },
+        "metadata.metadata.resolution_z_value": {
+            "max": 1500,
+            "min": 0,
+            "interval": 75
+        },
+        "metadata.metadata.rnaseq_assay_input": {
+            "max": 270000,
+            "min": 407,
+            "interval": 13479
+        },
+        "metadata.metadata.rnaseq_assay_input_value": {
+            "max": 1,
+            "min": 1,
+            "interval": 1
+        },
+        "metadata.metadata.roi_id": {
+            "max": 6,
+            "min": 1,
+            "interval": 1
+        },
+        "metadata.metadata.sc_isolation_cell_number": {
+            "max": 33500000,
+            "min": 1488,
+            "interval": 1674925
+        },
+        "metadata.metadata.sequencing_phix_percent": {
+            "max": 20,
+            "min": 0,
+            "interval": 1
+        },
+        "metadata.metadata.sequencing_read_percent_q30": {
+            "max": 96,
+            "min": 0,
+            "interval": 4
+        },
+        "metadata.metadata.step_z_value": {
+            "max": 2366,
+            "min": 598,
+            "interval": 88
+        },
+        "metadata.metadata.transposition_input": {
+            "max": 4000000,
+            "min": 3000,
+            "interval": 199850
+        }
+    },
+    "donor": {
+        "mapped_metadata.age_value": {
+            "max": 78,
+            "min": 1,
+            "interval": 3
+        },
+        "mapped_metadata.body_mass_index_value": {
+            "max": 50,
+            "min": 16,
+            "interval": 1
+        },
+        "mapped_metadata.height_value": {
+            "max": 193,
+            "min": 68,
+            "interval": 6
+        },
+        "mapped_metadata.kidney_donor_profile_index_value": {
+            "max": 100,
+            "min": 11,
+            "interval": 4
+        },
+        "mapped_metadata.weight_value": {
+            "max": 158,
+            "min": 10,
+            "interval": 7
+        }
+    },
+    "sample": {
+        "donor.mapped_metadata.age_value": {
+            "max": 78,
+            "min": 1,
+            "interval": 3
+        },
+        "donor.mapped_metadata.body_mass_index_value": {
+            "max": 50,
+            "min": 16,
+            "interval": 1
+        },
+        "donor.mapped_metadata.height_value": {
+            "max": 193,
+            "min": 68,
+            "interval": 6
+        },
+        "donor.mapped_metadata.kidney_donor_profile_index_value": {
+            "max": 100,
+            "min": 11,
+            "interval": 4
+        },
+        "donor.mapped_metadata.weight_value": {
+            "max": 158,
+            "min": 10,
+            "interval": 7
+        }
+    }
+}
```

Comment:
Does this go away down the road? I think you convinced me that it would be better to determine the bounds at query-time.

Reply:
Yes it is removed in #2661.